packages/databricks-vscode/resources/whats-new/2.4.md

```diff
@@ -0,0 +1,35 @@
+<img src="./1.2/databricks-logo.png" alt="databricks-logo" width="100"/>
+
+# Databricks Extension v2.4.0
+
+## What's new?
+
+-   [Databricks Asset Bundles support](#dabs)
+-   [Bundle configuration](#dabs-config)
+-   [No environment variables in terminals](#no-env)
+
+## <a id="dabs"></a> Databricks Asset Bundles support
+
+The extension now supports [Databricks Asset Bundles](https://docs.databricks.com/en/dev-tools/bundles/index.html)! New features include:
+
+-   Easy authentication and configuration of your Databricks Asset Bundles through the Visual Studio Code UI, including **AuthType** profile selection.
+-   A **Target** selector in the Databricks extension panel to quickly switch between bundle target environments.
+-   The **Override Jobs cluster** in bundle option in the extension panel to enable easy cluster override.
+-   A **Bundles Resource Explorer** view, which allows you to browse your bundle resources using the Visual Studio Code UI, deploy your local Databricks Asset Bundle’s resources to your remote Databricks workspace with a single click, and go directly to your deployed resources in your workspace from Visual Studio Code.
+-   A **Bundles Variables View**, which enables you to browse and edit your bundle variables using the Visual Studio Code UI. See Bundle Variables View.
+
+See [our documentation for more information](https://docs..databricks.com/dev-tools/vscode-ext/bundles.html).
+
+## <a id="dabs-config"></a> Bundle configuration
+
+The extension now requires a bundle configuration to be present in the project.
+
+If you don't use Databricks Asset Bundles, the extension will ask you to "migrate" your project. This will only create a `databricks.yml` configuration file with your workspace url.
+
+If you used our extension for a project before, we will migrate it automatically.
```

Comment:
```suggestion
If your project is a Databricks Asset Bundles project, the extension migrates it automatically.
```

Reply:
(I am opting not to, but I had the thought that we might want to explain that a DABs project is one with a databricks.yml.)

Reply:
I'm actually referring to the non-dabs projects for which uses used v1 extension. In that case we already have information about the host and auth, so we create dabs config automatically. Maybe I should rephrase it somehow to make it more clear? (or omit it completely)

When the projects already has DABs config, we don't do any additional migrations